README.md

```diff
@@ -13,6 +13,31 @@ We recommend using a link to a zip file of a [specific release](https://github.c
 
 https://defold.com/extension-websocket/api/
 
+## Additional server API reference
+
+This extension also includes a simple websocket server (ws://). Websocket server is not supported in HTML5 builds. Secure (wss://) websocket server is not supported.
```

Comment:
Why is that?
What is required for this to be the case?

Reply:
Secure web socket requires SSLSocket to be able to listen and the http_server to be able to work with it.
And I don't know how to implement that or how much work does it require. I know it should involve mbedtls and working with SSL certificates but it was beyond the scope of this project.